packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -935,7 +943,7 @@ class TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
         cursor: widget.mouseCursor ?? MouseCursor.defer,
         child: Listener(
           onPointerDown: _handlePointerDown,
-          behavior: HitTestBehavior.opaque,
+          behavior: HitTestBehavior.deferToChild, // Changed from translucent to deferToChild
```

Comment:
```suggestion
          behavior: HitTestBehavior.deferToChild,
```
Comments like this don't need to persist, years from now it won;t be relevant. :)